packages/terminal/src/browser/terminal-frontend-module.ts

```diff
@@ -52,9 +52,9 @@ export default new ContainerModule(bind => {
             const child = new Container({ defaultScope: 'Singleton' });
             child.parent = ctx.container;
             const counter = terminalNum++;
-            const domId = options.id || 'terminal-' + counter;
```

Comment:
Is it correct? I thought `id` can be provided by a client if it is responsible for managing it? 

It looks like after this change `id` is not used anymore breaking any client expecting it.

Should not we rather fix a client which provides duplicated ids?

Reply:
In the following commit 1731661, the option.id is kept